tests/e2e/pkg/tester/tester.go

```diff
@@ -203,6 +199,22 @@ func (t *Tester) addNodeIG() error {
 			ig = v
 		}
 	}
+	numNodes := int(*ig.Spec.MaxSize) // we assume that MinSize = Maxsize, this is true for e2e testing
```

Comment:
This will be the size of the last nodes InstanceGroup. Should we instead sum the max size of all nodes InstanceGroups?

Reply:
No, the ginkgo wants the number of all nodes except the master IG. When we are using kops for e2e testing we assume there are exactly 2 IGs(one master and one worker). If we add the master ig to the count, you'll have an extra node that you can't schedule workloads onto without tweaking manifests to tolerate control plane taints

Reply:
Ah I was thinking in the case of there being multiple nodes IGs. This tester code can be used in cluster configurations that have multiple node IGs so I don't know if your assumption is always true.